docs/function-instructions.md

```diff
@@ -337,6 +346,7 @@ If more than five genres are present, only the top five should be returned.
 ```javascript
 mostCommonGenres(books);
```

Comment:
To be consistent with how we are naming the other functions (i.e., starting with a verb), maybe we can call this function `getMostCommonGenres()`?